config/wpcalypso.json

```diff
@@ -111,6 +111,7 @@
 		"onboarding/import-light": false,
 		"onboarding/import-light-url-screen": true,
 		"onboarding/import-redirect-to-themes": true,
+		"onboarding/import-redesign": true,
```

Comment:
**comment:blocking** - one thing I should clarify here that I missed in the initial review is that this flag only affects `wpcalypso.wordpress.com`. 

Why does this matter? Because the E2E suite hits different hosts depending on the build configuration.

The specs
- `test/e2e/specs/onboarding/start__importer.ts`
- `test/e2e/specs/onboarding/setup__importer.ts` 

belong to the `calypso-pr` group, which is run as the Calypso E2E (desktop/mobile) builds. However, these builds use the `calypso.live` host, not `wpcalypso.wordpress.com`. 

In essence, turning this flag on and then checking it later in the specs is always going to evaluate to "false".